package.json

```diff
@@ -252,8 +255,10 @@
   },
   "homepage": "https://github.com/brave/brave-browser#readme",
   "dependencies": {
+    "adm-zip": "^0.4.13",
     "commander": "^2.9.0",
-    "fs-extra": "^1.0.0"
+    "fs-extra": "^1.0.0",
+    "sync-request": "^6.0.0"
```

Comment:
If i'm not mistaken, these can be devDependencies instead of dependencies. (`npm install --save-dev`)